### ISSUE TITLE
Design and execute documentation "portal" for Viz docs

### ISSUE THREAD

**Issue by stichbury** [kedro-org/kedro-viz#1524]
## Description
The parent ticket over on https://github.com/kedro-org/kedro/issues/2600 aims to move documentation for Kedro's packages into their own RTD subprojects. This means the docs can be released/updated alongside the packages rather than tied to the Framework cadence. We are particularly keen to do this for datasets because of ongoing issues with docstrings in the datasets code, but it also makes sense for Viz to have a degree of separation.

We aren't quite ready to divorce Kedro-Viz docs from Framework docs yet, but we need to plan what they'd look like when we do. There will be a sphinx project inside Viz and build process similar to that for Kedro, that builds docs for each release. 

We would then move Viz markdown from the "Visualisation" section into the Viz repo and build a set of docs based on what's there already plus anything else we have to offer such as the layout engine markdown that currently sits in this repo (and separate contribution guidelines for front end devs).

Additional work would also be needed to revise links from Viz docs -> Framework and Framework -> Viz.

## TL;DR
This ticket tracks the effort to define the new content architecture, set up a Sphinx project, execute the move and clean up the links.

**Comment by stichbury** [comment]
This is in progress: there's a branch open here https://github.com/kedro-org/kedro-viz/tree/new-docs-project where I'm working through the steps to create a new docs project

**Comment by stichbury** [comment]
Adding @astrojuanlu to this ticket for visibility and because I'll probably hand over for review of the Sphinx setup code.

**Comment by stichbury** [comment]
There's now a PR open in draft and I'm handing to @astrojuanlu for a week or so until I'm back from PTO https://github.com/kedro-org/kedro-viz/pull/1586

**Comment by tynandebold** [comment]
@stichbury #1586 is merged. Can we close this ticket?

**Comment by stichbury** [comment]
@tynandebold I think the answer is that we need to keep this ticket open to make the changes requested/recommended by @stephkaiser in her design review in https://github.com/kedro-org/kedro-viz/issues/1592

* Change the top left hand column (top of table of contents) to the yellow diamond + kedro rather than read "kedro-viz latest"
* Add example Kedro-Viz graphic to top of the main page (to replace the yellow diamond + kedro that is there presently)

**Comment by stichbury** [comment]
I can do both of these changes in the coming sprint.

**Comment by stichbury** [comment]
This has now merged and I've created a ticket for handover tasks: https://github.com/kedro-org/kedro-viz/issues/1637

**Comment by astrojuanlu** [comment]
Look at _that_ https://docs.kedro.org/projects/kedro-viz/
